src/dialog/reimu.c

```diff
@@ -61,13 +61,12 @@ DIALOG_TASK(reimu, Stage1PreBoss) {
 	MSG(cirno, "What, are you *that* impressed by my incredible magic?!");
```

Comment:
To be honest, I don't really like this whole interaction, especially Cirno's lines. Think you can give a shot at rewriting it?